cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1345,6 +1345,11 @@ EOF
   if [[ "${ENABLE_DEFAULT_STORAGE_CLASS:-}" == "true" ]]; then
     setup-addon-manifests "addons" "storage-class/gce"
   fi
+  if [[ "${ENABLE_METADATA_PROXY:-}" == "true" ]]; then
```

Comment:
Why is this a different env var than the gci script?

Reply:
Because I copy-pasta'd it wrong.  Done.